### PR TITLE
Remove Page Builder dependency

### LINKED ISSUE #935: Loader stays over the media gallery if it was opened second time from different control on the same page
### Preconditions (*)
1. Install Magento and Adobe Stock Integration [See documentation](https://github.com/magento/adobe-stock-integration/wiki/Installation-Guide)

### Steps to reproduce (*)
1. Login to Admin panel
2. Open Media Gallery (i.e. Catalog -> Category -> expand Content -> Select from Gallery button)
3. Close the Media Gallery
4. Open the Media Gallery using different entry point on the same page (i.e. WYSIWYG)

### Expected result (*)
Media Gallery is loaded and loader disappears

### Actual result (*)
Media Gallery is covered by loader that does not disappear

### LINKED ISSUE #1180: Remove Page Builder dependency
ASI should not depend on PB thus `\Magento\MediaGalleryIntegration\Plugin\UpdateOpenDialogUrlPageBuilder` should be removed.

Requires additional changes in PR repo.

### PR DESCRIPTION
**Pull request by Nazar65**
### Related Pull Requests
https://github.com/magento/magento2-page-builder/pull/450
https://github.com/magento/magento2/pull/27566

### Description (*)

This PR removes dependency between PageBuilder and MediaGallery
PageBuilder after merge https://github.com/magento/magento2-page-builder/pull/450 PB will use default Magento config, and anyone will be able to replace it

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1180: Remove Page Builder dependency
2. ...

### Manual testing scenarios (*)
Verify that Page Builder opens New Media Gallery without plugin

### CODE REVIEW

**Review by orlangur — DISMISSED** [review]
Functional tests failures does not seem to be related to changes introduced as to me.

**Comment by Nazar65** [comment]
integration tests will be passed after merge https://github.com/magento/magento2/pull/27863

**Review by lenaorobei — DISMISSED** [review]

File: MediaGalleryIntegration/Model/OpenDialogUrlProvider.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     *
+     * @return DataObject
+     */
+    public function getUrl(): string
+    {
+        if ($this->config->isEnabled()) {
+            return 'media_gallery/index/index';
+        }
+
+        return 'cms/wysiwyg_images/index';
```
Comment (coderimus) [inline MediaGalleryIntegration/Model/OpenDialogUrlProvider.php]: What about changing an `if` construction to the next one:
```php
return $this->config->isEnabled() ? `media_gallery/index/index` :  `cms/wysiwyg_images/index`
```

File: MediaGalleryIntegration/Model/OpenDialogUrlProvider.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     */
+    public function __construct(ConfigInterface $config)
+    {
+        $this->config = $config;
+    }
+
+    /**
+     * Get Url based on media gallery configuration
+     *
+     * @return DataObject
```
Comment (coderimus) [inline MediaGalleryIntegration/Model/OpenDialogUrlProvider.php]: Please, change the return type to the `string`.

**Review by coderimus — COMMENTED** [review]
Hi @Nazar65 thank you for this PR! Please, check my comments below.

**Comment by engcom-Golf** [comment]
@magento run all tests

**Comment by engcom-Golf** [comment]
@magento run all tests

**Comment by engcom-Golf** [comment]
@magento run all tests

**Review by lenaorobei — DISMISSED** [review]

**Comment by engcom-Golf** [comment]
@magento run all tests

**Pull request merged**